docs/src/topics/disks/data-disks.md

```diff
@@ -0,0 +1,70 @@
+# Data Disks
+This section describes how to specify additional data disks for a linode instance. These disks can use devices `sdb` through `sdh` 
+for a total of 7 disks. The total size of these disks + the OS Disk cannot exceed the linode plan size.
```

Comment:
Should probably add a note for how to check the plan size (https://www.linode.com/pricing/#compute-shared)